app/views/unattended/provisioning_templates/snippet/disk_enc_clevis_tang.erb

```diff
@@ -0,0 +1,89 @@
+<%#
+kind: snippet
+name: disk_enc_clevis_tang
+model: ProvisioningTemplate
+snippet: true
+description: |
+  Binds encrypted root directory ('/') utilizing Clevis to Tang server(s) for
+  decryption. The first parent device containing a LUKS container will be used.
+  The temporary passphrase will be removed afterwards. Currently, only Red Hat
+  family and Ubuntu operating systems are supported.
+-%>
+<%
+  passphrase = host_param('disk_enc_passphrase') || 'linux'
+  tang_server_list = []
+  packages_redhat = "clevis clevis-luks clevis-systemd clevis-dracut"
+  packages_ubuntu = "clevis clevis-luks clevis-systemd clevis-initramfs"
+
+  unless host_param('disk_enc_tang_servers').blank?
+    if host_param('disk_enc_tang_servers').is_a?(String)
+      tang_server_list = [host_param('disk_enc_tang_servers')]
+    else
+      tang_server_list = host_param('disk_enc_tang_servers')
+    end
+  end
+-%>
+
+<% if (@host.operatingsystem.family == 'Redhat' or @host.operatingsystem.name == 'Ubuntu') and unless tang_server_list.blank? %>
```

Comment:
Would this work for Debian / SuSE, too?

Reply:
For SUSE it doesn't work out-of-the-box and the host fails to boot up (probably something is missing in initrd). Debian was not tested.